test/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Create an executable program vox-map-test
+add_executable(vox-map-test VoxelMapTest.cpp)
+# The test program uses the shared library, use the pkg-config file
+pods_use_pkg_config_packages(vox-map-test occ-map)
```

Comment:
Yeah... just going to leave that there for a bit.  All will be cleansed in the long run, though.